src/add-to-project.ts

```diff
@@ -152,11 +224,37 @@ export async function addToProject(): Promise<void> {
       }`,
       {
         projectId,
-        title: issue?.html_url
-      }
+        title: issue?.html_url,
+      },
     )
 
     core.setOutput('itemId', addResp.addProjectV2DraftIssue.projectItem.id)
+    itemId = addResp.addProjectV2DraftIssue.projectItem.id
+  }
+
+  // update the project with the custom fields
+  for (const [key, value] of Object.entries(results)) {
+    const updateResp = await octokit.graphql<ProjectUpdateItemResponse>(
+      `mutation updateProjectItem($input: UpdateProjectV2ItemInput!) {
+          updateProjectV2Item(
+            input: {
+              projectId: $projectId,
+              itemId: $itemId,
+              fieldId: $fieldId,
+              value: $value
+          )
+        }`,
+      {
+        input: {
+          projectId,
+          itemId,
+          fieldId: key,
+          value,
+        },
+      },
+    )
+
+    core.setOutput('itemId', updateResp.updateProjectV2ItemById.item.id)
```

Comment:
@akalipetis I don't know why, but this block of code does not run

Reply:
@phemmylintry when you have the time, please push the code that we debugged together 💪🏽

Reply:
@akalipetis I've pushed the update